doc/release/release_dev.rst

```diff
@@ -98,6 +133,29 @@ Documentation
 - Update reference link documentation in the ``adjust_sigmoid`` function.
 - Fix reference to multiscale_basic_features in TrainableSegmenter.
 - Slight ``shape_index`` docstring modification to specify 2D array.
+- Add stitching gallery example (gh-5365)
+- Add draft SKIP3: transition to scikit-image 1.0 (gh-5475)
+- Mention commit messages in the contribution guidelines. (gh-5504)
+- Fix and standardize docstrings for blob detection functions (gh-5547)
+- Updated the User Guide to reflect usage of ``channel_axis`` rather than
+  ``multichannel``. (gh-5554)
+- Update the user guide to use channel_axis rather than multichannel (gh-5556)
+- Add hyperlinks to referenced documentation places. (gh-5560)
+- Update branching instructions to change the location of the pooch repo
+  (gh-5565)
+- add Notes and References section to the Cascade class docstring (gh-5568)
+- Clarify 2D vs nD in skimage.feature.corner docstrings (gh-5569)
```

Comment:
Not sure, because now there's a sequence of items ending with no period (just like a standard commit message)... :confused: